cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "caller_events": [],
```

Comment:
Randomness has decided that "caller_events" should always come first in the test files now